broker-util/oo-admin-ctl-domain

```diff
@@ -335,11 +335,14 @@ when "add_member"
   role = "admin" if role.nil?
   domain.add_members(user, Role.for(role))
   begin
-    domain.save
+    unless domain.save
+      puts "An error occurred during domain modification. Errors: #{domain.errors.messages}"
+      exit 3
+    end
     domain.run_jobs
-  rescue
+  rescue Exception => e
     puts "Unable to add \"#{user.login}\" to domain \"#{domain.canonical_namespace}\" with role \"#{role}\"."
-    puts "Errors: #{domain.errors.messages}"
```

Comment:
how about we check if #{domain.errors.messages} is empty and only then print "Error: #{e.message}" ?